apis/v1alpha2/object_reference_types.go

```diff
@@ -36,3 +36,34 @@ type LocalObjectReference struct {
 	// +kubebuilder:validation:MaxLength=253
 	Name string `json:"name"`
 }
+
+// ObjectReference identifies an API object including its namespace.
+type ObjectReference struct {
+	// Group is the group of the referent.
+	// When empty, the "core" API group is inferred.
```

Comment:
So interestingly, when the field is empty, it doesn't show up in 'kubectl get'. Just leaving a note here. No action required.